linux/meshd-nl80211.c

```diff
@@ -1303,6 +1306,9 @@ meshd_parse_libconfig (struct config_setting_t *meshd_section,
     CONFIG_LOOKUP(mcast-rate, mcast_rate);
     CONFIG_LOOKUP(beacon-interval,beacon_interval);
 
+    config_setting_lookup_int(meshd_section, "protected-management-frames",
+            (config_int_t *) &config->protected_management_frames);
```

Comment:
I changed it to "pmf" both in the code and config - if no objections I'll merge that.

RE encryption -- see https://kernel.googlesource.com/pub/scm/linux/kernel/git/jberg/mac80211-next/+/46f6b06050b736dab4d41494dae27b883cddc365